src/main/java/task/DukeException.java

```diff
@@ -0,0 +1,7 @@
+package task;
+
+public class DukeException extends Exception {
```

Comment:
This is a custom exception. Need to tell the users of your codebase what it is for => needs Javadoc comments